Assets/Scripts/Interaction Scripts/ControllerState.cs

```diff
@@ -8,15 +9,19 @@
 
 	/* This script is a wrapper around VRTK, to be used as a library for
 	higher-level scripts needing access to user input coming from a controller.
+	Initialize the ControllerState class as a singleton of the GameObject Controller.
 	For further clarifications, ask Apollo.
 	*/
```

Comment:
Please use the <summary> tag, it'll show up in auto-generated documentation too!